indico/modules/rb_new/controllers/backend/bookings.py

```diff
@@ -276,9 +276,40 @@ def _check_access(self):
         if not self.booking.can_edit(session.user):
             raise Forbidden
 
+    def _should_split(self, new_start_dt, new_end_dt):
+        today = date.today()
+        is_ongoing_booking = new_start_dt.date() < today
+        old_start_time, old_end_time = self.booking.start_dt.time(), self.booking.end_dt.time()
```

Comment:
I think repetition changes should be taken into account as well (it may result in some past occurrences being canceled otherwise) - or we could just disallow this type of change for an ongoing booking, it doesn't seem to make much sense to change the repetition mode there anyway, does it?

cc @pferreir

Reply:
hmm OTOH, if someone made a booking for a weekly meeting and then they switch to biweekly being able to apply this change for future occurrences seems sensible. maybe safer (and also easier i guess) to keep it and just trigger splitting for any repetition change